src/main/resources/morden-application.conf

```diff
@@ -0,0 +1,147 @@
+  keys-file = ${etc-client.datadir}"nodeId.keys"
+
+  keystore-dir = ${etc-client.datadir}"keystore"
+
+  // time the system will wait to shutdown the ActorSystem.
+  shutdown-timeout = "15.seconds"
+
+  # one of the algorithms defined here: https://docs.oracle.com/javase/8/docs/technotes/guides/security/StandardNames.html#SecureRandom
+  secure-random-algo = "NativePRNG"
+
+  network {
+
+    protocol-version = "1"
+
+    server-address {
+      interface = "127.0.0.1"
+      port = 9076
+    }
+
+    discovery {
+      bootstrap-nodes = [
+        "enode://e731347db0521f3476e6bbbb83375dcd7133a1601425ebd15fd10f3835fd4c304fba6282087ca5a0deeafadf0aa0d4fd56c3323331901c1f38bd181c283e3e35@128.199.55.137:30303",
+        "enode://ceb5c0f85eb994dbe9693bf46d99b03f6b838d17cc74e68d5eb003171ff39e5f120b17f965b267c319303f94d80b9d994b77062fb1486d76ce95d9f3d8fe1cb4@46.101.122.141:30303",
+        "enode://fb28713820e718066a2f5df6250ae9d07cff22f672dbf26be6c75d088f821a9ad230138ba492c533a80407d054b1436ef18e951bb65e6901553516c8dffe8ff0@104.155.176.151:30304",
+        "enode://afdc6076b9bf3e7d3d01442d6841071e84c76c73a7016cb4f35c0437df219db38565766234448f1592a07ba5295a867f0ce87b359bf50311ed0b830a2361392d@104.154.136.117:30403",
+        "enode://21101a9597b79e933e17bc94ef3506fe99a137808907aa8fefa67eea4b789792ad11fb391f38b00087f8800a2d3dff011572b62a31232133dd1591ac2d1502c8@104.198.71.200:30403"
+      ]
+
+      bootstrap-nodes-scan-interval = 2 minutes
+    }
+
+    peer {
+      connect-retry-delay = 20 seconds
+      connect-max-retries = 30
+      disconnect-poison-pill-timeout = 5 seconds
+      wait-for-hello-timeout = 3 seconds
+      wait-for-status-timeout = 30 seconds
+      wait-for-chain-check-timeout = 15 seconds
+
+      max-blocks-headers-per-message = 200
+      max-blocks-bodies-per-message = 200
+      max-receipts-per-message = 200
+      max-mpt-components-per-message = 400
+
+      max-peers = 10
+      network-id = 2
+    }
+
+    rpc {
+      enabled = true
+      interface = "127.0.0.1"
+      port = 8546
+      apis = "eth,web3,net"
+    }
+  }
+
+  txPool {
+    tx-pool-size = 1000
+    pending-tx-manager-query-timeout = 5.seconds
+  }
+
+  mining {
+    tx-pool-size = 1000
+    ommers-pool-size = 30
+    block-cashe-size = 30
+    coinbase = "0011223344556677889900112233445566778899"
+    pooling-services-timeout = 5.seconds
+  }
+
+  blockchain {
+    frontier-block-number = "0"
+    homestead-block-number = "494000"
+    eip150-block-number = "1783000" //todo add EIP155 ??
+    eip160-block-number = "1915000"
+    difficulty-bomb-pause-block-number = "1915000"
+    difficulty-bomb-continue-block-number = "3415000"
+
+    // Doc: https://blog.ethereum.org/2016/07/20/hard-fork-completed/
+    dao-fork-block-number = "1783000"
+    dao-fork-block-total-difficulty = "131072"//todo ?????? check that
+    dao-fork-block-hash = "f376243aeff1f256d970714c3de9fd78fa4e63cf63e32a51fe1169e375d98145"
+
+    account-start-nonce = "1048576"
```

Comment:
ok, so my answers was maybe not clear, only reason to add this is because it is in spec and we cannot connect to network if we set nonce according to YP
this change is needed because someone decided to set initial account nonce to different value when setting up morden